packages/gatsby/src/redux/types.ts

```diff
@@ -215,6 +215,7 @@ export interface IGatsbyState {
     trackedQueries: Map<Identifier, IQueryState>
     trackedComponents: Map<string, IComponentState>
     deletedQueries: Set<Identifier>
+    dirtyQueriesListToEmitViaWebsocket: Array<string>
```

Comment:
We can live with this for now, but I would like to address it in the follow-up someday 😄

Reply:
Just to make sure - this is about having this in this redux state slice or the type (array not set)?